README.md

```diff
@@ -110,7 +110,7 @@ foreman start
 
 The application will be running on port `5000`.
 
-You can launch an interactive console (ala `rails c`) using `racksh`.
+You can launch an interactive console (aka `rails c`) using `racksh`.
```

Comment:
There is typo, but it was intended to be `a la` not `ala`: https://en.wiktionary.org/wiki/a_la#English :)